crates/biome_diagnostics/tests/snapshots/html_mdx_print_diagnostic.snap

```diff
@@ -2,4 +2,4 @@
 source: crates/biome_diagnostics/tests/disagnostics.rs
 expression: content
 ---
-code&#95;block check <span style="color: #000; background-color: #ddd;"> FIXABLE </span> ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━<br /><br /><strong><span style="color: Tomato;">  </span></strong><strong><span style="color: Tomato;">✖</span></strong> <span style="color: Tomato;">This is the message of the diagnostic. It will appear in different colours based on the severity of the diagnostic.</span><br />  <br />
+code&#95;block:1:11 check <span style="color: #000; background-color: #ddd;"> FIXABLE </span> ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━<br /><br />  <strong><span style="color: Tomato;">✖</span></strong> <span style="color: Tomato;">This is the message of the diagnostic. It will appear in different colours based on the severity of the diagnostic.</span><br />  <br />  <strong><span style="color: Tomato;">&gt;</span></strong> <strong>1 │ </strong>fn main() &#123;<br />   <strong>   │ </strong>          <strong><span style="color: Tomato;">^</span></strong><br />  <strong><span style="color: Tomato;">&gt;</span></strong> <strong>2 │ </strong>    println!(&quot;Hello, world!&quot;);<br />   <strong>   │ </strong>    <strong><span style="color: Tomato;">^</span></strong><strong><span style="color: Tomato;">^</span></strong><strong><span style="color: Tomato;">^</span></strong><strong><span style="color: Tomato;">^</span></strong><br />    <strong>3 │ </strong>&#125;<br />  <br />
```

Comment:
Notice here, after the first two `<br /><br />` that you can find after the long line. We have two whitespace. They aren't covered by any special tag. Instead, on the left, we have `<strong><span style="color: Tomato;"> </span></strong>`